catalog/rest/rest.go

```diff
@@ -1159,55 +1159,95 @@ type createViewRequest struct {
 	Schema      *iceberg.Schema    `json:"schema"`
 	Location    string             `json:"location,omitempty"`
 	Props       iceberg.Properties `json:"properties,omitempty"`
```

Comment:
`properties` is marked as required field in the spec so we should remove the `omitempty`

https://github.com/apache/iceberg/blob/main/open-api/rest-catalog-open-api.yaml#L3556-L3560